dds/DCPS/transport/framework/TransportReassembly.h

```diff
@@ -97,17 +108,22 @@ class OpenDDS_Dcps_Export TransportReassembly {
   struct FragInfo {
     FragInfo()
       : complete_(false), have_first_(false), range_list_(), total_frags_(0) {}
```

Comment:
Do you need to initialize expiration_ here?

Reply:
I think it's OK defaulted to 0.  Any time we have a default constructed FragInfo it's immediately overwritten by an assignment from a FragInfo will all details.